docs/Getting-Started-frontend.md

```diff
@@ -170,7 +169,7 @@ For doing things like:
 
 - Setting up a local development environment with servers to view your website, power a backend API, and automatically refreshing your browser anytime files are modified
 - Watching files for changes and performing actions automatically (like syntax-checking/"linting" them, combining them, copying them, etc.)
-- Converting files from one format into another (bundling `.vue` and `.ejs` files into `.js`, etc.)
+- Converting files from one format into another (bundling `.vue` files into `.js`, etc.)
```

Comment:
Note: @taoeffect since we don't use `.ejs` anymore I thought it would be better to remove all references to it to avoid any confusion from future contributors. Is that ok for you?

Reply:
Yep!